images/win/scripts/Installers/Install-Vsix.ps1

```diff
@@ -0,0 +1,20 @@
+###################################################################################
+##  File:  Install-Vsix.ps1
+##  Desc:  Install the Visual Studio Extensions from toolset.json
+###################################################################################
+
+$ErrorActionPreference = "Stop"
+
+$toolset = Get-ToolsetContent
+$vsixPackagesList = $toolset.visualStudio.vsix
+if(-not $vsixPackagesList) {
+    Write-Host "No extensions to install"
+    exit 0
+}
+
+$VsVersion = $toolset.visualStudio.Version

```

Comment:
nitpick: upper-case previous was lower-case

Reply:
all 'V' has been made in upper case

Reply:
case of 'v' has been fixed